File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    let answer = 0;
    let arr = [];
    
    for(let i=0; i<nums.length-2; i++){
        for(let j=i+1; j<nums.length-1; j++){
            for(let k=j+1; k<nums.length; k++){
                arr.push(nums[i]+nums[j]+nums[k]);
            }
        }
    }
    
    for(let i=0; i<arr.length; i++){
    let cnt =0;
        for(let j=1; j<=arr[i]; j++){
            if(arr[i]%j === 0){
               cnt++; 
                if(cnt > 2) break;
            }
        }
        if(cnt === 2){
               answer++;
           }
    }
    
    
    return answer;
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    answer = 0
    for i in range(len(nums)):
        for j in range(i+1,len(nums)):
            for k in range(j+1,len(nums)):
                target = nums[i] + nums[j] + nums[k]
                for m in range(2,target):
                    if target % m == 0: 
                        break
                else: 
                    answer +=1 
    return answer